### ISSUE TITLE
AutoComplete: selected element not visible on keyboard navigation

### ISSUE THREAD

**Issue by WolfgangKluge** [Megabit/Blazorise#3828]
**Describe the bug**
When autocomplete shows a somewhat larger list (= there's a scrollbar), you can select those items with the arrow keys.
But after selecting an element that is out of view this way, the selection is not brought into view.

**To Reproduce**
1. Go to https://blazorise.com/docs/extensions/autocomplete
2. Type "a" in the first example
3. Select an element out of view (e.g. Netherlands Antilles) with the arrow keys
4. The value has changed, the element is selected - but the selected item is not visible

**Expected behavior**
The selected item should always be visible (scrollIntoView)
